### PR TITLE
feature/caf_plane_vectors

### PR DESCRIPTION
**Pull request by cjbackhouse**
Lose the `_planeN` variables and just use the vectors.

I /think/ these were in response to the difficulty of reading nested vectors in the CAFs, but flatCAF uptake is now high so no-one will miss them. I may be wrong though. We should do a quick survey.

A related question is whether these should be `vector<float>`, or `float[3]` or `array<float, 3>`

### CODE REVIEW

**Comment by cjbackhouse** [comment]
This is based on the units branch, so the diff should look more sensible once that is merged.

**Comment by etyley** [comment]
To be consistent, should this be extended to these?https://github.com/SBNSoftware/sbnanaobj/blob/f4576a50abb4af19a45152cb8ba699546092fa54/sbnanaobj/StandardRecord/SRTrack.h#L49-L54

**Comment by cjbackhouse** [comment]
> To be consistent, should this be extended to these?
> 
> https://github.com/SBNSoftware/sbnanaobj/blob/f4576a50abb4af19a45152cb8ba699546092fa54/sbnanaobj/StandardRecord/SRTrack.h#L49-L54

Ah yes. I missed them because they don't have `_plane` in their names.

**Comment by cjbackhouse** [comment]
I updated the PRs to also address SRTrack variables.

**Comment by cjbackhouse** [comment]
@gputnam will this cause any intolerable issues for python analysis? Everyone on the call today agreed they were using flatcafs so this is all win for the CAFAna analysis.

File: sbnanaobj/StandardRecord/SRTrack.cxx
Comment (etyley) [inline sbnanaobj/StandardRecord/SRTrack.cxx:11]: Rather than having this defined in multiple places, would it be preferable to have it defined somewhere common? (e.g. SREnums)

**Review by etyley — APPROVED** [review]
This all looks good to me, I have a stylistic comment that should not prevent this being merged.

File: sbnanaobj/StandardRecord/SRTrack.cxx
Comment (cjbackhouse) [reply sbnanaobj/StandardRecord/SRTrack.cxx:11]: I think SREnums is the wrong place, but I basically agree. Perhaps this ties in with the comment elsewhere about where `-5` should be defined.

Comment (cjbackhouse) [reply sbnanaobj/StandardRecord/SRTrack.cxx:11]: I filed https://github.com/SBNSoftware/sbnanaobj/issues/24

**Comment by cjbackhouse** [comment]
I'm discovering some subtleties this uncovers in SRProxy. Please don't merge this quite yet.

**Comment by cjbackhouse** [comment]
SRProxy issues resolved (the sbnana PR updates the srproxy version), but holding this as a draft at Gray's request so he can comment on Monday.

**Pull request merged**
